templates/hello-world-ts/README.md

```diff
@@ -0,0 +1,5 @@
+# Typescript Hello World
```

Comment:
Tiny issue - TypeScript should have a capital S

Reply:
fixed.